Mathlib/Algebra/BigOperators/Group/Finset/Defs.lean

```diff
@@ -102,9 +102,9 @@ open Batteries.ExtendedBinder Lean Meta
 /-- A `bigOpBinder` is like an `extBinder` and has the form `x`, `x : ty`, or `x pred`
 where `pred` is a `binderPred` like `< 2`.
 Unlike `extBinder`, `x` is a term. -/
-syntax bigOpBinder := term:max ((" : " term) <|> binderPred)?
```

Comment:
Isn't this whitespace considered good style?

Reply:
It may be considered good style, but the pretty-printer has not been made aware of this!  :smile:

Reply:
Perhaps for now the linter should skip `syntax` commands, since this needs core cooperation?